client/state/activity-log/log/schema.js

```diff
@@ -20,6 +21,7 @@ const activityItemSchema = {
 		activityGroup: { type: 'string' },
 		activityIcon: { type: 'string' },
 		activityId: { type: 'string' },
+		activityIsDiscarded: { type: 'boolean' },
```

Comment:
This is [always `null`](https://github.com/Automattic/wp-calypso/pull/18944/files#diff-f35c979bec454518e8757eb1864d776fR57) for the moment. This should be `oneOf: [ { type: 'boolean' }, { type: 'null' } ]` like `activityStatus` to validate correctly.

Reply:
Maybe it should be 
```js
{ type: [ 'boolean', 'null' ] }
```
This syntax has over 50 usages in Calypso.

The syntax with `oneOf` is only used twice in Calypso.

Reply:
👍  didn't know about that.

> The syntax with oneOf is only used twice in Calypso.

Maybe both mine 🤣